core/src/main/resources/reference.conf

```diff
@@ -170,12 +170,10 @@ kanaloa {
     # Metrics report configuration
     metrics {
       enabled = off
-      #  statsd {
-      #     host = "localhost"
+      #  use statsD for this metrics, also need to config statsD {host and port } down below
+      #  statsD {
```

Comment:
These are here because they are per dispatcher, e.g. you want to control `eventSampleRate` as a dispatcher by dispatcher basis. It's also under a statsD group here because I don't know if these settings make sense for another types of reporter.